src/app/shared/example-viewer/example-viewer.ts

```diff
@@ -59,20 +65,42 @@ export class ExampleViewer {
   private _example: string;
 
   /** Range of lines of the source code to display in compact view. */
-  @Input() lines: readonly [number, number];
+  @Input() region: string;
```

Comment:
I think this should be optional.
```suggestion
  @Input() region?: string;
```